schema/CompoundSchemaEnum.mcf

```diff
@@ -2504,3 +2504,88 @@ name: "DarpinAntibody "
 description: "DARPins (an acronym for designed ankyrin repeat proteins) are genetically engineered antibody mimetic proteins typically exhibiting highly specific and high-affinity target protein binding."
 descriptionUrl: "https://en.wikipedia.org/wiki/DARPin"
 
+Node: dcid:RelationshipAssociationTypeEnum
```

Comment:
I wonder if it's better to name this as RelationshipEvaluationResult? It's interesting to say that a relationship is type of "not associated", since "not associated" means "no relationship"?

Reply:
I think either works